### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by DerMolly**
### Component/Part
CONTRIBUTING.md

### Description
This PR adds CONTRIBUTING.md file.

### Steps

- [ ] added implementation
- [ ] added / updated tests
- [x] added / updated documentation
- [ ] extended changelog

### Related Issue(s)
#70

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+## How to contribute to HedgeDoc react-client
+
+#### Do you have questions about the project?
+
+* Ask any question about HedgeDoc in our [Matrix Support Channel][matrix-support] or our [Discourse][discourse].
```
Comment (mrdrogdrog) [inline CONTRIBUTING.md:5]:
```suggestion
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+* Feel free to join our [Matrix Support Channel][matrix-support] or post your question on our [Discourse][discourse].
+
+#### Did you find a bug?
+
+* **Ensure the bug was not already reported** by searching on GitHub under [Issues][issues].
```
Comment (mrdrogdrog) [inline CONTRIBUTING.md:10]:
```suggestion
* **Ensure the bug wasn't already reported** by searching on GitHub under [Issues][issues].
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+* Suggest your idea in the [HedgeDoc Dev Channel][matrix-dev] and start writing code. Our maintainers and other project developers can provide useful details about the architecture and show you relevant issues and discussions.
+
+
+HedgeDoc is a volunteer effort. We encourage you to pitch in and to help us making this project even better.
+
+Thanks! :heart: :heart: :heart:
+
+[issues]: https://github.com/codimd/react-client/issues
+[new_issue]: https://github.com/codimd/react-client/issues/new/choose
+[matrix-support]: https://app.element.io/#/room/#codimd:matrix.org
```
Comment (ErikMichelson) [inline CONTRIBUTING.md:31]: The main address of the room has changed too.

```suggestion
[matrix-support]: https://app.element.io/#/room/#hedgedoc:matrix.org
```

File: CONTRIBUTING.md (hunk 1 above)
Comment (DerMolly) [reply CONTRIBUTING.md:5]: what's the problem here? I don't understand.

**Comment by DerMolly** [comment]
Changes discussed with @ErikMichelson just now:
- [ ] a begining (something like 'you want to help? Great! Thanks! Here's how it works:')
- [ ] a mention of your translation efforts at POEditor

File: CONTRIBUTING.md (hunk 1 above)
Comment (DerMolly) [reply CONTRIBUTING.md:7]: If you're looking for the actual https-links they are in the last few lines of the file…

Comment (mrdrogdrog) [reply CONTRIBUTING.md:7]: You can delete this bullet point. The second one is better.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
+## How to contribute to HedgeDoc react-client
+
+Thanks for your interest in contributing. Here are some common scenarios for what you may want to contribute to.
+
+#### Do you have questions about the project?
+
+* Feel free to join our [Matrix Support Channel][matrix-support] or post your question on our [Discourse][discourse].
```
Comment (SISheogorath) [inline CONTRIBUTING.md:7]: I would recommend to put discourse first as answers there will allow to be found by search engines making everyone's life easier.

Comment (DerMolly) [reply CONTRIBUTING.md:7]: done

**Pull request merged**